recipes/gcc/all/conandata.yml

```diff
@@ -2,3 +2,9 @@ sources:
   "10.2.0":
     sha256: 27e879dccc639cd7b0cc08ed575c1669492579529b53c9ff27b0b96265fa867d
     url: https://ftp.gnu.org/gnu/gcc/gcc-10.2.0/gcc-10.2.0.tar.gz
+  "11.3.0":
```

Comment:
newer can go on top please 🙏

Reply:
Can do but other packages I looked at are in version order aren't they?

Reply:
https://github.com/conan-io/conan-center-index/blob/4bf114eefecf85163341d07233fa94b6186ae97c/docs/package_templates/cmake_package/config.yml#L2

They should be ordered but just the opposite direction  😅